Python/test/ratehelpers.py

```diff
@@ -730,9 +730,6 @@ def testFxBasisSwapsWithCollateralAndBasisInQuoteCcy(self):
         self.assertImpliedQuotes(
             is_fx_base_ccy_collateral_ccy, is_basis_on_fx_base_currency_leg)
 
-    def tearDown(self):
-        ql.Settings.instance().evaluationDate = ql.Date()
```

Comment:
No, I would keep this — the problem was elsewhere.

Reply:
Ok, I will revert this change.